helm/panda/charts/jedi/sandbox/init-jedi-db.py

```diff
@@ -0,0 +1,29 @@
+import time
```

Comment:
This script is not required since those queries are executed in [ /post_step_panda.sql](https://github.com/PanDAWMS/panda-database/blob/main/initdb/sqls/post_step_panda.sql).
In any case, all database-related things should go to panda-database or panda-k8s/helm/panda/charts/postgres.